.github/workflows/release-notes.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         id: extract_branch
         run: echo "value=${GITHUB_REF:11}" >> $GITHUB_OUTPUT
       # If it has release drafter:
-      - uses: release-drafter/release-drafter@v5
+      - uses: release-drafter/release-drafter@v6
```

Comment:
There are no breaking changes in this release:
https://github.com/release-drafter/release-drafter/releases/tag/v6.0.0